yt_dlp/extractor/paramountplus.py

```diff
@@ -90,7 +90,7 @@ class ParamountPlusIE(CBSBaseIE):
     def _extract_video_info(self, content_id, mpx_acc=2198311517):
         items_data = self._download_json(
             'https://www.paramountplus.com/apps-api/v2.0/androidtv/video/cid/%s.json' % content_id,
-            content_id, query={'locale': 'en-us', 'at': 'ABCqWNNSwhIqINWIIAG+DFzcFUvF8/vcN6cNyXFFfNzWAIvXuoVgX+fK4naOC7V8MLI='})
+            content_id, query={'locale': 'en-us', 'at': 'ABCqWNNSwhIqINWIIAG+DFzcFUvF8/vcN6cNyXFFfNzWAIvXuoVgX+fK4naOC7V8MLI='}, headers=self.geo_verification_headers())
```

Comment:
This commit fixed the issue!

```bash
root@000 ~/Downloads/YT_SNAP/yt-dlp-paramountplus-fix2 # yt-dlp --ignore-config --geo-verification-proxy PROXY_IP:PROXY_PORT --cookies /STORAGE/PARAMOUNT/PPLUS_COOKIE.txt --verbose 'https://www.paramountplus.com/shows/the-stand/video/JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ/the-stand-the-end/' -N50
[debug] Command-line config: ['--ignore-config', '--geo-verification-proxy', 'PROXY_IP:PROXY_PORT', '--cookies', '/STORAGE/PARAMOUNT/PPLUS_COOKIE.txt', '--verbose', 'https://www.paramountplus.com/shows/the-stand/video/JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ/the-stand-the-end/', '-N50']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] yt-dlp version 2021.08.02
[debug] Python version 3.7.3 (CPython 64bit) - Linux-4.19.0-17-amd64-x86_64-with-debian-10.10
[debug] exe versions: ffmpeg v1.27-6-ge0542c4, ffprobe v1.27-6-ge0542c4, phantomjs 1.9.2, rtmpdump 2.4
[debug] Optional libraries: keyring, mutagen, pycryptodome, sqlite, websockets
[debug] Proxy map: {}
[debug] [ParamountPlus] Extracting URL: https://www.paramountplus.com/shows/the-stand/video/JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ/the-stand-the-end/
[ParamountPlus] JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ: Downloading JSON metadata
[ParamountPlus] JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ: Downloading JSON metadata
[ParamountPlus] JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ: Downloading DASH_CENC_PRECON SMIL data
[ParamountPlus] JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ: Downloading m3u8 information
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, vcodec:vp9.2(10), acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ: Downloading 1 format(s): hls-4547
[debug] Invoking downloader on "https://cbsios-vh.akamaihd.net/i/temp_hd_gallery_video/CBS_Production_Outlet_VMS/video_robot/CBS_Production_Entertainment/2020/12/05/1828043843952/CBS_THE_STAND_101_AA_8CH_394714_,2228,4628,3128,1628,848,503,000.mp4.csmil/index_1_av.m3u8?null=0&id=AgCU+wd9AhC6ZLPNGWE72jOPjwJaVsE4O6orvqhhiDeoaTeYwXe7+hXODex8BDLD7dr8guMkOmOqXQ%3d%3d&hdntl=exp=1629167411~acl=%2fi%2ftemp_hd_gallery_video%2fCBS_Production_Outlet_VMS%2fvideo_robot%2fCBS_Production_Entertainment%2f2020%2f12%2f05%2f1828043843952%2fCBS_THE_STAND_101_AA_8CH_394714_*~data=hdntl~hmac=e718b6a6109a1e9fa7e328fb74eedfcc884e14db8a0da4c0c2c281398094058c"
[hlsnative] Downloading m3u8 manifest
[hlsnative] Total fragments: 354
[download] Destination: The Stand - The End [JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ].mp4
WARNING: The download speed shown is only of one thread. This is a known issue and patches are welcome
[download] 100% of 1.93GiB in 00:24
[debug] ffmpeg command line: ffprobe -show_streams 'file:The Stand - The End [JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ].mp4'
[FixupM3u8] Fixing malformed AAC bitstream of "The Stand - The End [JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ].mp4"
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:The Stand - The End [JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ].mp4' -c copy -map 0 -dn -f mp4 -bsf:a aac_adtstoasc 'file:The Stand - The End [JudCDITcmIS5TjyvnKdqjY5zHNR1t9wZ].temp.mp4'
[SponSkrub] Skipping sponskrub since it is not a YouTube video
```